src/frontend/views/Attester/instagram.svg

```diff
@@ -0,0 +1 @@
top stop-color="#FF543E" offset="50%"/><stop stop-color="#C837AB" offset="100%"/></radialGradient><radialGradient cx="-17.076%" cy="7.771%" fx="-17.076%" fy="7.771%" r="44.326%" id="prefix__d"><stop stop-color="#3771C8" offset="0%"/><stop stop-color="#3771C8" offset="13%"/><stop stop-color="#60F" stop-opacity="0" offset="100%"/></radialGradient><path id="prefix__a" d="M26.042 2.778l5.472 15.035L21.229 30.07 5.472 27.292 0 12.257 10.285 0z"/><path id="prefix__f" d="M0 14V0h1.74v14z"/><filter x="-143.7%" y="-17.9%" width="502.3%" height="150%" filterUnits="objectBoundingBox" id="prefix__e"><feOffset dx="1" dy="1" in="SourceAlpha" result="shadowOffsetOuter1"/><feGaussianBlur stdDeviation="1" in="shadowOffsetOuter1" result="shadowBlurOuter1"/><feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.5 0" in="shadowBlurOuter1"/></filter></defs><g fill="none" fill-rule="evenodd"><path fill="#FFF" d="M31.213 4.313l6.499 17.854-12.213 14.555-18.712-3.3L.289 15.57 12.502 1.014z"/><g transform="translate(3.243 3.833)"><mask id="prefix__b" fill="#fff"><use xlink:href="#prefix__a"/></mask><use fill="red" xlink:href="#prefix__a"/><g mask="url(#prefix__b)" fill-rule="nonzero"><path d="M18.999 0C11.075 0 8.769 0 8.304.047 6.676.18 5.674.438 4.56.988a7.573 7.573 0 00-2.184 1.596 8.145 8.145 0 00-2.2 4.625c-.13.877-.167 1.07-.176 5.607V19c0 7.928 0 10.247.047 10.697.134 1.584.38 2.581.906 3.671a8.012 8.012 0 005.185 4.23c.906.217 1.831.34 2.762.366.47.02 5.261.035 10.063.035 4.803 0 9.602 0 10.058-.03a13.02 13.02 0 002.859-.373 7.956 7.956 0 005.188-4.248 9.39 9.39 0 00.897-3.601c.026-.328.035-5.555.035-10.761 0-5.207 0-10.425-.035-10.752a9.294 9.294 0 00-.915-3.637 7.55 7.55 0 00-1.628-2.227A8.183 8.183 0 0030.801.175C29.924.047 29.734.01 25.192 0H19z" fill="url(#prefix__c)" transform="translate(-3.243 -3.833)"/><path d="M18.999 0C11.075 0 8.769 0 8.304.047 6.676.18 5.674.438 4.56.988a7.573 7.573 0 00-2.184 1.596 8.145 8.145 0 00-2.2 4.625c-.13.877-.167 1.07-.176 5.607V19c0 7.928 0 10.247.047 10.697.134 1.584.38 2.581.906 3.671a8.012 8.012 0 005.185 4.23c.906.217 1.831.34 2.762.366.47.02 5.261.035 10.063.035 4.803 0 9.602 0 10.058-.03a13.02 13.02 0 002.859-.373 7.956 7.956 0 005.188-4.248 9.39 9.39 0 00.897-3.601c.026-.328.035-5.555.035-10.761 0-5.207 0-10.425-.035-10.752a9.294 9.294 0 00-.915-3.637 7.55 7.55 0 00-1.628-2.227A8.183 8.183 0 0030.801.175C29.924.047 29.734.01 25.192 0H19z" fill="url(#prefix__d)" transform="translate(-3.243 -3.833)"/></g></g><path d="M13.626 4.104L29.1 6.832l5.374 14.764-9.916 11.817-5.454-14.668L3.71 15.921l9.915-11.817z" stroke-opacity=".5" stroke="#FFF" stroke-width=".5" fill-opacity=".248" fill="#FFF"/><path stroke-opacity=".601" stroke="#FFF" stroke-width=".5" stroke-linecap="round" stroke-linejoin="round" d="M29.033 6.936l-9.988 11.726"/><g fill-rule="nonzero"><g transform="translate(17.99 11.865)"><use fill="#000" filter="url(#prefix__e)" xlink:href="#prefix__f"/><use fill="#FFF" xlink:href="#prefix__f"/></g></g></g></svg>
```

Comment:
We always use svgo to optimize our SVG images, using the command `npx svgo [path to SVG image]`